index.html

```diff
@@ -4,484 +4,528 @@
-                  </h2>
-                  <p class="cards-content__middle_subtitle">
-                    Full Time - Singapore
-                  </p>
-                </div>
+              <!-- card content middle -->
+              <div class="card-content__middle">
+                <p class="card-content__title">Project Manager</p>
+                <p class="card-content__subtitle">Full Time - Singapore</p>
+              </div>
 
-                <!-- Cards content bottom -->
-                <div class="cards-content__bottom">
-                  <a class="cards-content__button grey-btn" href="#">
-                    <p class="cards-content__button-text">Remote</p>
-                  </a>
-                  <a class="cards-content__button grey-btn" href="#">
-                    <p class="cards-content__button-text">Anywhere</p>
-                  </a>
-                  <p class="cards-content__button-text black-txt">
-                    $10k - $25k
-                  </p>
-                </div>
+              <!-- card content bottom -->
+              <div class="card-content__bottom">
+                <span class="card-content__feature" href="#"> Remote </span>
+                <span class="card-content__feature" href="#"> Anywhere </span>
+                <p class="feature__text">$10k - $25k</p>
               </div>
-            </div>
+            </a>
           </div>
 
           <!-- Scroll block -->
-          <a class="first-section__scroll-block" href="#">
-            <h2 class="scroll-block__text">View All</h2>
-            <img src="./src/icons/arrow.svg" alt="arrow" />
+          <a class="job-posts__scroll-block" href="#">
+            <h2 class="scroll-block__text">
+              View All
+              <svg
+                width="28"
+                height="18"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M.875 9a.937.937 0 0 1 .938-.938h22.111l-5.9-5.898A.938.938 0 1 1 19.35.836l7.5 7.5a.937.937 0 0 1 0 1.328l-7.5 7.5a.938.938 0 1 1-1.327-1.328l5.9-5.899H1.813A.938.938 0 0 1 .875 9Z"
+                />
+              </svg>
+            </h2>
           </a>
         </div>
       </section>
 
       <!-- Second section -->
-      <section class="second-section">
-        <div class="container second-section__container">
-          <h2 class="second-section__title title">Job Categories</h2>
-          <div class="second-section__jobs-categories">
-            <!-- Job cards -->
-            <div class="jobs-group first-jobs-group">
-              <a href="#" class="content-container__job-blocks job-block">
+      <section class="categories">
+        <div class="container categories__container">
+          <h2 class="categories__title title">Job Categories</h2>
+          <div class="categories__jobs-categories">
```

Comment:
```suggestion
          <div class="jobs-categories-wrapper">
```
I see now that it would be better to use a "block" naming